src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -117,9 +116,6 @@ public function load(array $configs, ContainerBuilder $container)
 
         $bundles = $container->getParameter('kernel.bundles');
         if (isset($bundles['SecurityBundle'])) {
-            if (class_exists(ExpressionLanguage::class)) {
-                $loader->load('security_expression_language.xml');
```

Comment:
Actually, `symfony/expression-language` is in `require-dev` of `symfony/security-bundle`: https://github.com/symfony/symfony/blob/b5c6892911fc7454a76528cb0d4d1af4336c7413/src/Symfony/Bundle/SecurityBundle/composer.json#L42

So we still need to check that the `ExpressionLanguage` component is actually available.

Reply:
Are you sure? `security.expression_language` would not be present so the alias would become invalid and as we use `on-invalid="null"`  on `api_platform.security.resource_access_checker` that should work. Haven't checked though.